demo/README.md

```diff
@@ -0,0 +1 @@
+# Blobstream rollup dev environment
```

Comment:
```suggestion
# Blobstream rollup dev environment

To run this demo, first clone the repository:

```bash
cd $HOME
git clone https://github.com/celestiaorg/orchestrator-relayer.git
cd orchestrator-relayer/demo
```

Then run the docker-compose setup:

```bash
docker-compose up -d
```

Reply:
this should also have closing code backticks ```